renin/src/renin.ts

```diff
@@ -638,6 +643,13 @@ export class Renin {
     if (this.options.productionMode) {
       return false;
     }
+    localStorage.setItem(
```

Comment:
How often is this function called – will this localStorage add a noticeable penalty on the update loop in dev mode? 🤔